src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -105,6 +105,55 @@
     <BitButton Target="_self" Href="https://github.com/bitfoundation/bitframework" IsEnabled="false">
         <span>Bit Foundation From Span</span>
     </BitButton>
+
+    <hr />
+
+    <div>
+        <h3>Toggle Button</h3>
+        <h4>Checked Button</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")></BitToggleButton>
+
+        <h4>Toggle Button with Onclick event </h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         OnClick="ToggleButtonClick"></BitToggleButton>
+        <span>Check status is: @OnToggleButtonChecked</span>
+
+        <h4>Toggle Button with Standard style</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         ButtonStyle="ButtonStyle.Standard"></BitToggleButton>
+
+        <h4>Toggle Button with Primary style</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         ButtonStyle="ButtonStyle.Primary"></BitToggleButton>
+
+        <h4>Disabeled Toggle Buttons</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         IsEnabled="false"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")></BitToggleButton>
+
+
+        <h4>Toggle Button with aria description</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         AriaDescription="Detailed description used for screen reader."></BitToggleButton>
+
+        <h4>Toggle Button with aria hidden</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         AriaHidden="true"></BitToggleButton>
+    </div>
+
```

Comment:
please introduce another sample in which the 2-way parameter gets changed from outside of the component